.gitlab-ci.yml

```diff
@@ -31,15 +31,16 @@ stages:
 
 # some default values
 variables:
-  # Format: bionic_coq-V$DATE-$hash
+  # Format: image_name-V$DATE-$hash
   # $DATE is so we can tell what's what in the image list
   # The $hash is the first 10 characters of the md5 of the Dockerfile. e.g.
-  # echo $(md5sum dev/ci/docker/bionic_coq/Dockerfile | head -c 10)
-  CACHEKEY: "bionic_coq-V2023-06-29-2b9dca3152"
-  IMAGE: "$CI_REGISTRY_IMAGE:$CACHEKEY"
+  # echo $(md5sum dev/ci/docker/old_ubuntu_lts/Dockerfile | head -c 10)
+  # echo $(md5sum dev/ci/docker/edge_ubuntu/Dockerfile | head -c 10)
+  BASE_CACHEKEY: "old_ubuntu_lts-V2023-06-30-42bb810307"
```

Comment:
we use `base/edge` terminology on this file, maybe we could imitate that here?

Reply:
I know, but the "old LTS" terminology is on purpose, because it defines how the version of this Ubuntu image should be updated (always use the old LTS version, i.e., the one before the last LTS version).